components/tools/OmeroPy/src/omero/util/populate_metadata.py

```diff
@@ -1605,10 +1631,14 @@ def _write_to_omero_batch(self, to_delete, loops=10, ms=500):
         # an exception has been thrown (likely LockTimeout)
         rsp = callback.getResponse()
         if isinstance(rsp, omero.cmd.OK):
```

Comment:
This may never reach the logger because:

```
rsp <class 'omero.cmd.Delete2Response'> object #0 (::omero::cmd::Delete2Response)
{   
    deletedObjects =
    {   
        key = ome.model.annotations.WellAnnotationLink
        value =
        {   
            [0] = 8371438
        }
    }
}
```